addons/web/static/src/core/hotkeys/hotkey_service.js

```diff
@@ -114,21 +114,20 @@ export const hotkeyService = {
                     continue;
                 }
 
-                if (!reg.altIsOptional && !isAlted) {
-                    continue;
-                }
-
                 if (!reg.allowRepeat && event.repeat) {
                     continue;
                 }
 
                 reg.callback();
                 dispatched = true;
             }
-
-            if (!event.repeat && isAlted) {
-                // Click on all elements having a data-hotkey attribute matching the actual hotkey.
-                const elems = activeElement.querySelectorAll(`[data-hotkey='${hotkey}' i]`);
+            const overlayModifier = hotkeyService.overlayModifier.split("+");
```

Comment:
wrong naming IMO.
It is an array, thus needs an explicit plural concept.
Suggestion: overlayModParts ?

Reply:
You forgot to modify the name :-)

Reply:
Oh and this can't work unless the [FIX] commit comes after the [IMP] one :-)